internal/runbits/requirements/requirements.go

```diff
@@ -218,6 +222,18 @@ func (r *RequirementOperation) ExecuteRequirementOperation(
 
 		requirementName = normalized
 
+		// If a bare version number was given, and if it is a partial version number (e.g. requests@2),
+		// we'll want to ultimately append a '.x' suffix.
+		if versionRe.MatchString(requirementVersion) {
```

Comment:
nit: Any appetite to centralizing this logic that is shared between `init` and `install` now?

Reply:
No, they don't actually share much of the same code. One queries for a language, the other for a package/bundle.